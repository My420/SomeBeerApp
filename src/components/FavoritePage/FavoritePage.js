import React from 'react';
import styles from './FavoritePage.module.scss';
import Main from '../Main/Main';
import Container from '../Container/Container';
import Catalog from '../FavoriteCatalog/FavoriteCatalog';

const FavoritePage = () => {
  // eslint-disable-next-line no-console
  console.log('render ==================== FavoritePage');

  return (
    <Main className={styles.main}>
      <Container className={styles.container}>
        <section className={styles.favoritePage}>
          <h2 className="visually-hidden">Favorite</h2>
          <Catalog />
        </section>
      </Container>
    </Main>
  );
};

export default FavoritePage;
