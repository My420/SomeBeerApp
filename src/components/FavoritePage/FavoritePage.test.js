import React from 'react';
import { shallow } from 'enzyme';
import FavoritePage from './FavoritePage';

describe('test <FavoritePage /> component', () => {
  const wrapper = shallow(<FavoritePage />);

  test('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  test('should  render Connect(FavoriteWrapper(FavoriteCatalog)) component ', () => {
    expect(
      wrapper.find('Connect(FavoriteWrapper(FavoriteCatalog))')
    ).toHaveLength(1);
  });
});
