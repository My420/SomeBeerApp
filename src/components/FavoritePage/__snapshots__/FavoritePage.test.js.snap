// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test <FavoritePage /> component should match snapshot 1`] = `
<Main
  className="main"
>
  <Container
    className="container"
  >
    <section
      className="favoritePage"
    >
      <h2
        className="visually-hidden"
      >
        Favorite
      </h2>
      <Connect(FavoriteCatalog) />
    </section>
  </Container>
</Main>
`;
